README.md

```diff
@@ -72,23 +72,37 @@ wikipediaPreview.init({
 
 To indicate that a word or expression should bring up the article preview popup, mark it with the `data-wikipedia-preview` attribute (or anything else as long as you're using the `selector` option described above).
 
-By default, nodes with the `data-wikipedia-preview` attribute don't have any special visual treatment. You should style them in a way that makes sense for your context.
+#### data-wp-title
 
-Example
+When the article title is not the same as the node's `textContent` property, use the `data-wp-title` attribute to specify the article title.
+
+#### data-wp-lang
+
+To use a language different than the language specified in the options, use the `data-wp-lang` attribute.
+
+#### .wmf-wp-with-preview
+
+To use the default trigger link styling:
+
+* Add the following link to the page header:
+```html
+<link href="dist/wikipediaPreview.css" rel="stylesheet">
```

Comment:
The official doc shouldn't mention the `dist/` folder.

Reply:
One alternative is simply not including the HTML example and updating text to something like:

> Add a `<link>` in your page header referencing the `wikipediaPreview.css` asset

Just curious, why shouldn't the doc mention the `/dist` folder?

Reply:
I think the HTML example is fine. 

The dist/ folder makes sense for us in the source repo but when we distribute, only the compiled output should be provided.

Reply:
Hmm right, how about just removing `dist/` then and have it be consistent with what's under 'Standalone Script' section?

```html
<link href="wikipediaPreview.css" rel="stylesheet">
```